### DISCUSSION TITLE
Widgets and Taskbar Window Overlapping

### DISCUSSION THREAD

**Discussion by seth-aung** [zhongyang219/TrafficMonitor#1827] [General]
![image](https://github.com/zhongyang219/TrafficMonitor/assets/94091504/d277bf0e-d9b9-4bad-8625-461ff9f4d639)
After fresh install of windows 11 and several windows updates, widgets and taskbar window overlapping when I set the taskbar behaviors to the left.

**Comment by INTRASERVER** [comment]
![traffic monitor](https://github.com/zhongyang219/TrafficMonitor/assets/97578992/07a0354a-e8a7-426a-ab7e-d29143afe1d8)

It works very well. You have to look for unofficial update.

**Reply by seth-aung** [reply]
Yes. Not work well when you change taskbar behavior to left and you don't even have widget on your taskbar.

**Comment by INTRASERVER** [comment]
Try this one: 
https://github.com/liukyle/TrafficMonitor/releases/tag/V1.81.4

**Reply by ZYong9908** [reply]
This one is better, but it still doesn't display properly, especially when there is no window present
![image](https://github.com/user-attachments/assets/7b288313-5cf4-47cf-a221-be95c37e21a7)
![image](https://github.com/user-attachments/assets/fa9f8cc9-de92-4fb0-a506-7e4fee9fab07)

**Reply by DrMirage** [reply]
try

https://github.com/HowerZiu/TrafficMonitor/tree/4hdds
